dinky-admin/src/main/java/org/dinky/service/impl/UserServiceImpl.java

```diff
@@ -254,7 +254,7 @@ public Result<Void> assignRole(AssignRoleParams assignRoleParams) {
     }
 
     @Override
-    public Result<Tenant> chooseTenant(Integer tenantId) {
+    public Result chooseTenant(Integer tenantId) {
```

Comment:
heres needn't change

Reply:
Okay, I have made the necessary modifications.